makefile

```diff
@@ -1713,6 +1713,7 @@ shaders: bgfx-tools
 translation:
 	$(SILENT) echo Generating mame.pot
 	$(SILENT) find src -iname "*.cpp" | xargs xgettext --from-code=UTF-8 -k_ -k__ -o mame.pot
+	$(SILENT) find plugins -iname "*.lua" | xargs xgettext --from-code=UTF-8 -k_ -k__ -j -o mame.pot
```

Comment:
This should really use `-print0` and `xargs -0` to avoid potential problems with spaces.  Also you can use `-or` to do this with a single command.  It's probably better to make a separate rule for mame.pot since it isn't actually a phony target, and have the `translation` rule depend on that.  I might do it myself at some point.

Reply:
I don't feel familiar enough with makefiles to change the rules that much without messing it up.